io.itch.drwhut.TabletopClub.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "io.itch.drwhut.TabletopClub",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "tabletop-club",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--share=ipc",
+        "--device=dri",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=xdg-desktop/TabletopClub:ro",
+        "--filesystem=xdg-documents/TabletopClub:create",
+        "--filesystem=xdg-download/TabletopClub:ro"
```

Comment:
why why you need them `ro` if they are specific to this?

Reply:
This is how the standalone executable scans the file system for custom assets. It will only ever read from Desktop and Downloads if there are directories there, and it will create the ones in Documents to store save files and screenshots. I deduced this was the minimum permissions necessary to replicate the standalone version's behaviour.

Reply:
it doesn't make sense whatsoever.